pandas/tests/frame/methods/test_cov_corr.py

```diff
@@ -58,6 +59,24 @@ def test_cov(self, float_frame, float_string_frame):
         )
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize("test_ddof", [0, 1, 2, 3])
+    def test_cov_ddof(self, test_ddof):
+        np_array1 = np.random.rand(10)
+        np_array2 = np.random.rand(10)
+
+        s1 = Series(np_array1)
+        s2 = Series(np_array2)
+
+        result = s1.cov(s2, ddof=test_ddof)
+        expected = np.cov(np_array1, np_array2, ddof=test_ddof)[0][1]
+        assert math.isclose(expected, result)
```

Comment:
can you use assert_series_equal here

Reply:
it returns a float. So the test fails with assert_series_equal. Is there any other option or should I convert into a Series?

Reply:
don’t select out a single value
compare the result in its entirety

Reply:
Sorry, but I am struggling to see your point here.
In fact differently from Dataframe.cov, Series.cov returns a float and not the covariance matrix.
So the one compared is the entire result of Series.cov (float) with the covariance matrix value of s1,s2 (float).
Can you please help me to understand what you mean with entire result?